<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prowler</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_Amazon_view">
   <property name="icon">
    <iconset>
     <normalon>resources/Amazon-icon.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Amazon view...</string>
   </property>
   <property name="toolTip">
    <string>Open Amazon view</string>
   </property>
  </action>
  <action name="actionOpen_Vendor_view">
   <property name="icon">
    <iconset>
     <normalon>148705-essential-collection/png/list-1.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Vendor view...</string>
   </property>
   <property name="toolTip">
    <string>Open Vendor view</string>
   </property>
  </action>
  <action name="actionOpen_Operations_view">
   <property name="icon">
    <iconset>
     <normalon>148705-essential-collection/png/settings-2.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Operations view...</string>
   </property>
   <property name="toolTip">
    <string>Open Operations view</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
